service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorController.java

```diff
@@ -76,11 +110,22 @@
         }
     }
 
+    @ApiOperation(
+            value = "Creates or updates connector configururation for external subject attributes for the given zone.",
+            tags = { "Attribute Connector Management" },
+            response = AttributeConnector.class)
+    @ApiResponses(
+            value = { @ApiResponse(
+                    code = 201,
+                    message = "Connector configuration for the given zone is successfully created.") })
     @RequestMapping(
             method = PUT,
             value = V1 + AcsApiUriTemplates.SUBJECT_CONNECTOR_URL,
-            consumes = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<AttributeConnector> putSubjectConnector(@RequestBody final AttributeConnector connector) {
+            consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Good catch.  I changed return type from ResponseEntity<AttributeConnector> to ResponseEntity<?>